app/experimenter/experiments/migrations/0181_remove_non_ldap_emails.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.5 on 2021-08-05 20:41
+
+from django.db import migrations
+
+
+def remove_non_ldap_emails(apps, schema_editor):
+    User = apps.get_model("auth", "User")
+    User.objects.filter(
+        analyzed_experiments__isnull=True,
+        experimentchangelog__isnull=True,
+        experimentcomment__isnull=True,
+        nimbuschangelog__isnull=True,
+        notifications__isnull=True,
+        owned_experiments__isnull=True,
+        owned_nimbusexperiments__isnull=True,
+        subscribed_experiments__isnull=True,
+    ).exclude(email__icontains="mozilla").delete()
```

Comment:
pocket people would have @getpocket.com?

Reply:
I've tried asking the auth folks for the list of all valid ldap domains and I couldn't get anything.  Good catch @tiftran 🙏

Reply:
Also the fallback here is if they don't have any foreign keys, they get deleted, next time they access, account gets recreated, no data is lost.